prow/cmd/hook/trigger.go

```diff
@@ -176,9 +188,9 @@ func (ga *GitHubAgent) trustedPullRequest(pr github.PullRequest) (bool, error) {
 
 func (ga *GitHubAgent) askToJoin(pr github.PullRequest) error {
 	commentTemplate := `
-Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply with "@k8s-bot ok to test" on its own line.
+Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply with "@k8s-bot ok to test" on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands will still work. Regular contributors should join the org to skip this step.
 
-Regular contributors should join the org to skip this step.`
+If you have questions or suggestions related to this bot's behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository.`
```

Comment:
maybe link to the new issue page with a template?
like this:

https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:&body=%0A%0D/assign%20@spxtr